assets/js/googlesitekit/notifications/components/common/Description.js

```diff
@@ -28,7 +28,7 @@ import { Fragment } from '@wordpress/element';
  * Internal dependencies
  */
 import { sanitizeHTML } from '../../../../util';
-import ErrorText from '../../../../components/ErrorText';
+import ErrorNotice from '@/js/components/ErrorNotice';
```

Comment:
Unclear why this is better/we started doing it, as relative imports allow tooling like VSCode to automatically adjust paths when files move, etc.

What's the reason for this change? 🤔